app/src/getters/index.ts

```diff
@@ -21,3 +28,43 @@ export function getProjectRegistryContract(chainId: number, web3: Web3) {
 
     return new web3.eth.Contract(projectRegistryAbi as AbiItem[], contractAddress)
 }
+
+export async function getAllowance(
```

Comment:
This is the new version of the `getAllowance` utility. This one asks people to retry if things go south.